test/app_test.rb

```diff
@@ -26,4 +26,33 @@ def test_analysis
     expected = "{\"results\":[{\"type\":\"shebang\",\"keys\":[\"shebang\"]}]}"
     assert_equal expected, last_response.body
   end
+
+  def test_ignores_shebang_for_test_files
+    code = <<~CODE
+      #!/usr/bin/env ruby
+      require 'minitest/autorun'
+      require_relative 'exercise'
+
+      # Common test data version: 1.0.0 e9e9ee9
+      class    ExerciseTest    <    Minitest::Test
+      end
+    CODE
+    post "/analyze/ruby", {code: code}.to_json
+
+    expected = "{\"results\":[]}"
```

Comment:
Making the outside quotes into single quotes will mean you don't need to quote the double quotes.